tests/mock_tests/portsorch_ut.cpp

```diff
@@ -146,8 +146,13 @@ namespace portsorch_test
 
         ASSERT_EQ(gPortsOrch, nullptr);
 
-        gPortsOrch = new PortsOrch(m_app_db.get(), m_state_db.get(), ports_tables, m_chassis_app_db.get());
+        vector<string> flex_counter_tables = {
+            CFG_FLEX_COUNTER_TABLE_NAME
+        };
+        auto* flexCounterOrch = new FlexCounterOrch(m_config_db.get(), flex_counter_tables);
+        gDirectory.set(flexCounterOrch);
 
+	    gPortsOrch = new PortsOrch(m_app_db.get(), m_state_db.get(), ports_tables, m_chassis_app_db.get());
```

Comment:
Are you  using tab here? If so, please change it to white spaces. Otherwise please fix the indentation.

Reply:
Fixed

Reply:
What unit test is done with this new flexCounterOrch?

Reply:
This is to adapt to the new change, there is no new UT on this file.
Since PortsOrch object is calling functions on the FlexOrch this change was needed.
The UT added on a different python file.